carp.studies.core/src/commonMain/kotlin/dk/cachet/carp/studies/domain/users/ParticipantRepository.kt

```diff
@@ -0,0 +1,22 @@
+package dk.cachet.carp.studies.domain.users
+
+import dk.cachet.carp.common.UUID
+
+
+/**
+ * Store [Participant] instances, linked to study IDs.
+ */
+interface ParticipantRepository
```

Comment:
Nothing new here. Participants are now simply stored in a separate repository from `StudyRepository` since we can envision it becoming a separate microservice down the line.